cmd/zdb/zdb.c

```diff
@@ -5932,7 +5932,9 @@ zdb_leak_init_prepare_indirect_vdevs(spa_t *spa, zdb_cb_t *zcb)
 		vdev_metaslab_group_create(vd);
 		VERIFY0(vdev_metaslab_init(vd, 0));
 
+#if defined(DEBUG) && !defined(NDEBUG)
 		vdev_indirect_mapping_t *vim = vd->vdev_indirect_mapping;
```

Comment:
```suggestion
		vdev_indirect_mapping_t *vim __maybe_unused = vd->vdev_indirect_mapping;
```

Rather than wrapping this I'd suggest using `__maybe_unused` which we've defined in `lib/libspl/include/sys/debug.h` as the gcc/clang attribute for this purpose.

```c
#ifndef __maybe_unused
#define __maybe_unused __attribute__((unused))
#endif
```

Reply:
fixed.